### DISCUSSION TITLE
Programming virgin card megane 3

### DISCUSSION THREAD

**Discussion by Minatodu93** [cedricp/ddt4all#1164] [Q&A]
Dear All, 
 I wonder if you can help me for programming a new card/key for my megane 3 I search about in internet but didn’t found anything I tried by myself seraching in ddt4all in every type of UCH/ECU but didn’t found anything.
 Thank you

**Comment by matzel82** [comment]
Only at Original Garage. It have to connect to Renault Server.

work only with original keycard from Renault.

when NOT you can try to clone with ddt

**Comment by Minatodu93** [comment]
How can I clone it with DDT I didn’t found anything about
